tests/run/cpp_stl_cpp11.pyx

```diff
@@ -140,6 +140,11 @@ def test_unordered_set_functionality():
     return "pass"
 
 
+cdef extern from "cpp_unordered_map_helper.h":
```

Comment:
You can also just include the C++ code as a literal docstring here. That avoids having to look up another file when reading through the test.